cassiopeia/data/CassiopeiaTree.py

```diff
@@ -1116,6 +1116,30 @@ def depth_first_traverse_edges(
 
         return nx.dfs_edges(self.__network, source=source)
 
+    def breadth_first_traverse_edges(
+        self, source: Optional[int] = None
```

Comment:
I believe you mean `Optional[str]`, since nodes `str` in `CassiopeiaTree`?

Reply:
thanks!